analysis/tests/lighttpd-minimal/src/main.rs

```diff
@@ -183,8 +1292,14 @@ unsafe extern "C" fn fdnode_free(fdn: *mut libc::c_void) {
 }
 
 pub unsafe extern "C" fn lighttpd_test() {
-    let fdarr = malloc(::std::mem::size_of::<*mut fdnode>() as libc::c_ulong); // TODO: handle cast as *mut *mut fdnode;
-    let fdes = malloc(::std::mem::size_of::<fdevents>() as libc::c_ulong); // TODO: handle cast as *mut fdevents;
+    let fdarr = calloc(
+        0 as libc::c_int as libc::c_ulong,
```

Comment:
Can you just confirm these are supposed to be 0?  I remember we had a nullptr deref from something like this before.

Reply:
these are contrived for the time being, the actual code is more involved

Reply:
That's fine as long as we make sure to update them before they are used and not just immediately freed.